Remove unnecessary non_contiguous and gradient tests from test_linalg

lezcano opened the pull request:
Stack from [ghstack](https://github.com/ezyang/ghstack):
* #67833
* __->__ #68188
* #67996
* #68523
* #68522
* #69909
* #69908

As per title

cc @mruberry @jianyuh @nikitaved @pearu @walterddr @IvanYashchuk @xwang233 @Lezcano

Differential Revision: [D33774419](https://our.internmc.facebook.com/intern/diff/D33774419)

lezcano committed:
Remove unnecessary non_contiguous tests from test_linalg

As per title

[ghstack-poisoned]

lezcano committed:
Update on "Remove unnecessary non_contiguous tests from test_linalg"


As per title

cc mruberry jianyuh nikitaved pearu walterddr IvanYashchuk xwang233 Lezcano

[ghstack-poisoned]

lezcano committed:
Update on "Remove unnecessary non_contiguous tests from test_linalg"


As per title

cc mruberry jianyuh nikitaved pearu walterddr IvanYashchuk xwang233 Lezcano

[ghstack-poisoned]

lezcano committed:
Update on "Remove unnecessary non_contiguous tests from test_linalg"


As per title

cc mruberry jianyuh nikitaved pearu walterddr IvanYashchuk xwang233 Lezcano

[ghstack-poisoned]

mruberry reviewed (approved):
Cool!

Just ping me when @IvanYashchuk and @nikitaved have gotten through this stack, too ;)

lezcano committed:
Update on "Remove unnecessary non_contiguous tests from test_linalg"


As per title

cc mruberry jianyuh nikitaved pearu walterddr IvanYashchuk xwang233 Lezcano

[ghstack-poisoned]

lezcano committed:
Update on "Remove unnecessary non_contiguous tests from test_linalg"


As per title

cc mruberry jianyuh nikitaved pearu walterddr IvanYashchuk xwang233 Lezcano

[ghstack-poisoned]

IvanYashchuk reviewed (approved):
> +4 −317 

The stats of this PR look really great!

lezcano committed:
Update on "Remove unnecessary non_contiguous tests from test_linalg"


As per title

cc mruberry jianyuh nikitaved pearu walterddr IvanYashchuk xwang233 Lezcano

[ghstack-poisoned]

lezcano committed:
Update on "Remove unnecessary non_contiguous tests from test_linalg"


As per title

cc mruberry jianyuh nikitaved pearu walterddr IvanYashchuk xwang233 Lezcano

[ghstack-poisoned]

lezcano committed:
Update on "Remove unnecessary non_contiguous tests from test_linalg"


As per title

cc mruberry jianyuh nikitaved pearu walterddr IvanYashchuk xwang233 Lezcano

[ghstack-poisoned]

lezcano committed:
Update on "Remove unnecessary non_contiguous tests from test_linalg"


As per title

cc mruberry jianyuh nikitaved pearu walterddr IvanYashchuk xwang233 Lezcano

[ghstack-poisoned]

lezcano committed:
Update on "Remove unnecessary non_contiguous tests from test_linalg"


As per title

cc mruberry jianyuh nikitaved pearu walterddr IvanYashchuk xwang233 Lezcano

[ghstack-poisoned]

IvanYashchuk commented on test/test_linalg.py:
```diff
@@ -1124,6 +1084,7 @@ def test_eigvalsh_errors_and_warnings(self, device, dtype):
             with self.assertRaisesRegex(RuntimeError, "tensors to be on the same device"):
                 torch.linalg.eigvalsh(t, out=out)
 
+<<<<<<< HEAD
```
Git conflict markers have leaked into the PR.

Lezcano replied:
Oooops. Thanks for the catch, fixed now.

lezcano committed:
Update on "Remove unnecessary non_contiguous and gradient tests from test_linalg"


As per title

cc mruberry jianyuh nikitaved pearu walterddr IvanYashchuk xwang233 Lezcano

[ghstack-poisoned]

lezcano committed:
Update on "Remove unnecessary non_contiguous and gradient tests from test_linalg"


As per title

cc mruberry jianyuh nikitaved pearu walterddr IvanYashchuk xwang233 Lezcano

[ghstack-poisoned]

lezcano committed:
Update on "Remove unnecessary non_contiguous and gradient tests from test_linalg"


As per title

cc mruberry jianyuh nikitaved pearu walterddr IvanYashchuk xwang233 Lezcano

[ghstack-poisoned]

lezcano committed:
Update on "Remove unnecessary non_contiguous and gradient tests from test_linalg"


As per title

cc mruberry jianyuh nikitaved pearu walterddr IvanYashchuk xwang233 Lezcano

[ghstack-poisoned]

lezcano committed:
Update on "Remove unnecessary non_contiguous and gradient tests from test_linalg"


As per title

cc mruberry jianyuh nikitaved pearu walterddr IvanYashchuk xwang233 Lezcano

[ghstack-poisoned]

lezcano committed:
Update on "Remove unnecessary non_contiguous and gradient tests from test_linalg"


As per title

cc mruberry jianyuh nikitaved pearu walterddr IvanYashchuk xwang233 Lezcano

[ghstack-poisoned]

lezcano committed:
Update on "Remove unnecessary non_contiguous and gradient tests from test_linalg"


As per title

cc mruberry jianyuh nikitaved pearu walterddr IvanYashchuk xwang233 Lezcano

[ghstack-poisoned]

lezcano committed:
Update on "Remove unnecessary non_contiguous and gradient tests from test_linalg"


As per title

cc mruberry jianyuh nikitaved pearu walterddr IvanYashchuk xwang233 Lezcano

[ghstack-poisoned]

lezcano committed:
Update on "Remove unnecessary non_contiguous and gradient tests from test_linalg"


As per title

cc mruberry jianyuh nikitaved pearu walterddr IvanYashchuk xwang233 Lezcano

[ghstack-poisoned]

lezcano committed:
Update on "Remove unnecessary non_contiguous and gradient tests from test_linalg"


As per title

cc mruberry jianyuh nikitaved pearu walterddr IvanYashchuk xwang233 Lezcano

[ghstack-poisoned]

lezcano committed:
Update on "Remove unnecessary non_contiguous and gradient tests from test_linalg"


As per title

cc mruberry jianyuh nikitaved pearu walterddr IvanYashchuk xwang233 Lezcano

[ghstack-poisoned]

lezcano committed:
Update on "Remove unnecessary non_contiguous and gradient tests from test_linalg"


As per title

cc mruberry jianyuh nikitaved pearu walterddr IvanYashchuk xwang233 Lezcano

[ghstack-poisoned]

lezcano committed:
Update on "Remove unnecessary non_contiguous and gradient tests from test_linalg"


As per title

cc mruberry jianyuh nikitaved pearu walterddr IvanYashchuk xwang233 Lezcano

[ghstack-poisoned]

lezcano committed:
Update on "Remove unnecessary non_contiguous and gradient tests from test_linalg"


As per title

cc mruberry jianyuh nikitaved pearu walterddr IvanYashchuk xwang233 Lezcano

[ghstack-poisoned]

lezcano committed:
Update on "Remove unnecessary non_contiguous and gradient tests from test_linalg"


As per title

cc mruberry jianyuh nikitaved pearu walterddr IvanYashchuk xwang233 Lezcano

[ghstack-poisoned]

lezcano committed:
Update on "Remove unnecessary non_contiguous and gradient tests from test_linalg"


As per title

cc mruberry jianyuh nikitaved pearu walterddr IvanYashchuk xwang233 Lezcano

[ghstack-poisoned]

lezcano committed:
Update on "Remove unnecessary non_contiguous and gradient tests from test_linalg"


As per title

cc mruberry jianyuh nikitaved pearu walterddr IvanYashchuk xwang233 Lezcano

[ghstack-poisoned]

lezcano committed:
Update on "Remove unnecessary non_contiguous and gradient tests from test_linalg"


As per title

cc mruberry jianyuh nikitaved pearu walterddr IvanYashchuk xwang233 Lezcano

[ghstack-poisoned]

lezcano committed:
Update on "Remove unnecessary non_contiguous and gradient tests from test_linalg"


As per title

cc mruberry jianyuh nikitaved pearu walterddr IvanYashchuk xwang233 Lezcano

[ghstack-poisoned]

lezcano committed:
Update on "Remove unnecessary non_contiguous and gradient tests from test_linalg"


As per title

cc mruberry jianyuh nikitaved pearu walterddr IvanYashchuk xwang233 Lezcano

[ghstack-poisoned]

lezcano committed:
Update on "Remove unnecessary non_contiguous and gradient tests from test_linalg"


As per title

cc mruberry jianyuh nikitaved pearu walterddr IvanYashchuk xwang233 Lezcano

[ghstack-poisoned]

lezcano committed:
Update on "Remove unnecessary non_contiguous and gradient tests from test_linalg"


As per title

cc mruberry jianyuh nikitaved pearu walterddr IvanYashchuk xwang233 Lezcano

[ghstack-poisoned]

lezcano committed:
Update on "Remove unnecessary non_contiguous and gradient tests from test_linalg"


As per title

cc mruberry jianyuh nikitaved pearu walterddr IvanYashchuk xwang233 Lezcano

[ghstack-poisoned]

lezcano committed:
Update on "Remove unnecessary non_contiguous and gradient tests from test_linalg"


As per title

cc mruberry jianyuh nikitaved pearu walterddr IvanYashchuk xwang233 Lezcano

[ghstack-poisoned]

lezcano committed:
Update on "Remove unnecessary non_contiguous and gradient tests from test_linalg"


As per title

cc mruberry jianyuh nikitaved pearu walterddr IvanYashchuk xwang233 Lezcano

[ghstack-poisoned]

lezcano committed:
Update on "Remove unnecessary non_contiguous and gradient tests from test_linalg"


As per title

cc mruberry jianyuh nikitaved pearu walterddr IvanYashchuk xwang233 Lezcano

[ghstack-poisoned]

lezcano committed:
Update on "Remove unnecessary non_contiguous and gradient tests from test_linalg"


As per title

cc mruberry jianyuh nikitaved pearu walterddr IvanYashchuk xwang233 Lezcano

[ghstack-poisoned]

lezcano committed:
Update on "Remove unnecessary non_contiguous and gradient tests from test_linalg"


As per title

cc mruberry jianyuh nikitaved pearu walterddr IvanYashchuk xwang233 Lezcano

[ghstack-poisoned]

lezcano committed:
Update on "Remove unnecessary non_contiguous and gradient tests from test_linalg"


As per title

cc mruberry jianyuh nikitaved pearu walterddr IvanYashchuk xwang233 Lezcano

[ghstack-poisoned]

lezcano committed:
Update on "Remove unnecessary non_contiguous and gradient tests from test_linalg"


As per title

cc mruberry jianyuh nikitaved pearu walterddr IvanYashchuk xwang233 Lezcano

[ghstack-poisoned]

lezcano committed:
Update on "Remove unnecessary non_contiguous and gradient tests from test_linalg"


As per title

cc mruberry jianyuh nikitaved pearu walterddr IvanYashchuk xwang233 Lezcano

[ghstack-poisoned]

lezcano committed:
Update on "Remove unnecessary non_contiguous and gradient tests from test_linalg"


As per title

cc mruberry jianyuh nikitaved pearu walterddr IvanYashchuk xwang233 Lezcano

[ghstack-poisoned]

lezcano committed:
Update on "Remove unnecessary non_contiguous and gradient tests from test_linalg"


As per title

cc mruberry jianyuh nikitaved pearu walterddr IvanYashchuk xwang233 Lezcano

[ghstack-poisoned]

lezcano committed:
Update on "Remove unnecessary non_contiguous and gradient tests from test_linalg"


As per title

cc mruberry jianyuh nikitaved pearu walterddr IvanYashchuk xwang233 Lezcano

[ghstack-poisoned]

lezcano committed:
Update on "Remove unnecessary non_contiguous and gradient tests from test_linalg"


As per title

cc mruberry jianyuh nikitaved pearu walterddr IvanYashchuk xwang233 Lezcano

[ghstack-poisoned]

lezcano committed:
Update on "Remove unnecessary non_contiguous and gradient tests from test_linalg"


As per title

cc mruberry jianyuh nikitaved pearu walterddr IvanYashchuk xwang233 Lezcano

[ghstack-poisoned]

lezcano committed:
Update on "Remove unnecessary non_contiguous and gradient tests from test_linalg"


As per title

cc mruberry jianyuh nikitaved pearu walterddr IvanYashchuk xwang233 Lezcano

[ghstack-poisoned]

lezcano committed:
Update on "Remove unnecessary non_contiguous and gradient tests from test_linalg"


As per title

cc mruberry jianyuh nikitaved pearu walterddr IvanYashchuk xwang233 Lezcano

[ghstack-poisoned]

lezcano committed:
Update on "Remove unnecessary non_contiguous and gradient tests from test_linalg"


As per title

cc mruberry jianyuh nikitaved pearu walterddr IvanYashchuk xwang233 Lezcano

[ghstack-poisoned]

lezcano committed:
Update on "Remove unnecessary non_contiguous and gradient tests from test_linalg"


As per title

cc mruberry jianyuh nikitaved pearu walterddr IvanYashchuk xwang233 Lezcano

[ghstack-poisoned]

lezcano committed:
Update on "Remove unnecessary non_contiguous and gradient tests from test_linalg"


As per title

cc mruberry jianyuh nikitaved pearu walterddr IvanYashchuk xwang233 Lezcano

[ghstack-poisoned]

lezcano committed:
Update on "Remove unnecessary non_contiguous and gradient tests from test_linalg"


As per title

cc mruberry jianyuh nikitaved pearu walterddr IvanYashchuk xwang233 Lezcano

[ghstack-poisoned]

lezcano committed:
Update on "Remove unnecessary non_contiguous and gradient tests from test_linalg"


As per title

cc mruberry jianyuh nikitaved pearu walterddr IvanYashchuk xwang233 Lezcano

[ghstack-poisoned]

lezcano committed:
Update on "Remove unnecessary non_contiguous and gradient tests from test_linalg"


As per title

cc mruberry jianyuh nikitaved pearu walterddr IvanYashchuk xwang233 Lezcano

[ghstack-poisoned]

lezcano committed:
Update on "Remove unnecessary non_contiguous and gradient tests from test_linalg"


As per title

cc mruberry jianyuh nikitaved pearu walterddr IvanYashchuk xwang233 Lezcano

[ghstack-poisoned]

mruberry commented:
@mruberry has imported this pull request.  If you are a Facebook employee, you can view this diff [on Phabricator](https://www.internalfb.com/diff/D33774419).

lezcano committed:
Update on "Remove unnecessary non_contiguous and gradient tests from test_linalg"


As per title

cc mruberry jianyuh nikitaved pearu walterddr IvanYashchuk xwang233 Lezcano

Differential Revision: [D33774419](https://our.internmc.facebook.com/intern/diff/D33774419)

[ghstack-poisoned]

mruberry commented:
@mruberry has imported this pull request.  If you are a Facebook employee, you can view this diff [on Phabricator](https://www.internalfb.com/diff/D33774419).

lezcano committed:
Update on "Remove unnecessary non_contiguous and gradient tests from test_linalg"


As per title

cc mruberry jianyuh nikitaved pearu walterddr IvanYashchuk xwang233 Lezcano

Differential Revision: [D33774419](https://our.internmc.facebook.com/intern/diff/D33774419)

[ghstack-poisoned]

mruberry commented:
@mruberry has imported this pull request.  If you are a Facebook employee, you can view this diff [on Phabricator](https://www.internalfb.com/diff/D33774419).

lezcano committed:
Update on "Remove unnecessary non_contiguous and gradient tests from test_linalg"


As per title

cc mruberry jianyuh nikitaved pearu walterddr IvanYashchuk xwang233 Lezcano

Differential Revision: [D33774419](https://our.internmc.facebook.com/intern/diff/D33774419)

[ghstack-poisoned]

mruberry commented:
@mruberry has imported this pull request.  If you are a Facebook employee, you can view this diff [on Phabricator](https://www.internalfb.com/diff/D33774419).
